build_templates/mkmf.template.rttov.intel

```diff
@@ -129,29 +129,35 @@ LD = ifort
 
 # You will likely not need to modify below this line
 
-HDFINC = $(HDFDIR)/include
-HDFLIB = $(HDFDIR)/lib
-HDFLIBS = -lhdf5hl_fortran -lhdf5_hl -lhdf5_fortran -lhdf5 
+#HDFINC = /glade/u/home/thoar/hdf-eos/include
+#HDFLIB = /glade/u/home/thoar/hdf-eos/lib
+#HDFLIBS = -lhdfeos -lGctp -lmfhdf -ldf -ljpeg -lz
+
+#HDF5DIR = /glade/u/apps/ch/opt/hdf-eos5/5.1.16/intel/19.0.5
+#HDF5INC = $(HDF5DIR)/include
+#HDF5LIB = $(HDF5DIR)/lib
+#HDF5LIBS = -lhdf5hl_fortran -lhdf5_hl -lhdf5_fortran -lhdf5
```

Comment:
I'm looking for advice on how to handle the HDF dependencies across platforms ...